purchase_next_reception_date/models/purchase_order.py

```diff
@@ -0,0 +1,36 @@
+# Copyright 2023 Quartile Limited
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, fields, models
+
+
+class PurchaseOrder(models.Model):
+    _inherit = "purchase.order"
+
+    next_reception_date = fields.Date(
```

Comment:
I just realized that this field is defined as `Date` and it will cause an issue, for example, when the next receipt date according to the order lines is expected to be '2023-08-27 08:00:00' JST - the next reception date in the header will be '2023-08-26'.

Therefore, we should either make the field `Datetime`, or convert the datetime into date with `context_timestamp()` in the compute method.  From design perspective, we should go for the former, as the result of the latter will depend on user's timezone setting.